src/main/k/working/imp/prove/sum-div-spec.k

```diff
@@ -15,38 +15,36 @@ module SUM-DIV-SPEC
   imports VERIFICATION
 
 
-rule
+  rule
     <k>
       while (!(n <= 0)) {
-        sum = sum + n;
-        n = n + -1;
+          sum = sum + n;
+          n = n + -1;
       }
-    =>
-      .K
-    ...</k>
-  <state>
-      n   |-> (N:Int => 0)
-      sum |-> (S:Int => S +Int (((N +Int 1) *Int N) /Int 2))
-  </state>
-requires N >=Int 0
-
-rule
+      => .K
+      ...
+    </k>
+    <state>
+        n   |-> (N:Int => 0)
+        sum |-> (S:Int => S +Int (((N +Int 1) *Int N) /Int 2))
+    </state>
+  requires N >=Int 0
+
+  rule
     <k>
       int n, sum;
       n = N:Int;
       sum = 0;
       while (!(n <= 0)) {
-        sum = sum + n;
-        n = n + -1;
+          sum = sum + n;
+          n = n + -1;
       }
-    =>
-      .K
+      => .K
     </k>
-  <state>
-    .Map
-  =>
-      n   |-> 0
-      sum |-> (((N +Int 1) *Int N) /Int 2)
-  </state>
-requires N >=Int 0
+    <state>
+      .Map =>
```

Comment:
You can leave it as-is, but it seems harder to spot the arrow if it's on the same line as `.Map`. I'm not sure about others, but that's usually one of the first things I do when looking at a spec file.